fixtures/reconciler/README.md

```diff
@@ -0,0 +1,16 @@
+
+To test, run:
+* `yarn install` 
+* edit node_modules/react-reconciler/cjs/react-reconciler.development.js 
+* Add the below snipper after `var valueStack = [];`
+* `yarn test`
```

Comment:
It doesn't seem ideal that our test project involves editing content in `node_modules`. I would expect to see something more like:

```
yarn install
yarn build
yarn test
```

Where the `yarn build` step just runs `yarn build -- reconciler` in the main project to build a copy of the reconciler.

Then the test file (`index.js` I guess) should test something meaningful without any editing being required.

Reply:
Idk if this needs to stick around or how to write a test that ensures that each reconciler has its own scope for reconciler-unique things like `valueStack` other than this really hacky way.

Possibly something with actually creating two reconcilers and then making them both do work that would indicate if they were clobbering each other or not.

Reply:
Yes, we could automate it. At this point I don't think it's important if we manually verify it works once.